### ISSUE TITLE
imgui-rs v0.8 has been released and this crate seems to be incompatible

### ISSUE THREAD

**Issue by ChristianIvicevic** [Yatekii/imgui-wgpu-rs#72]
Just wanted to let you know that a new version of imgui-rs and the platform crates have been released and there is a version mismatch and incompatibility as expected:
```
error: failed to select a version for `imgui-sys`.
    ... required by package `imgui v0.7.0`
    ... which is depended on by `imgui-wgpu v0.17.0`
    ... which is depended on by `blight v0.1.0 (/Users/near/Code/blight)`
versions that meet the requirements `^0.7.0` are: 0.7.0

the package `imgui-sys` links to the native library `imgui`, but it conflicts with a previous package which links to `imgui` as well:
package `imgui-sys v0.8.0`
    ... which is depended on by `imgui v0.8.0`
    ... which is depended on by `blight v0.1.0 (/Users/near/Code/blight)`
Only one package in the dependency graph may specify the same links value. This helps ensure that only one copy of a native library is linked in the final binary. Try to adjust your dependencies so that only one package uses the links ='imgui-sys' value. For more information, see https://doc.rust-lang.org/cargo/reference/resolver.html#links.

failed to select a version for `imgui-sys` which could resolve this conflict
```

**Comment by cwfitzgerald** [comment]
Sorry about the delay, will get to this tonight.

**Comment by cwfitzgerald** [comment]
Published as imgui-wgpu 0.17.1
